spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ImportExportControllerHelperService.java

```diff
@@ -452,7 +452,7 @@ public JSONObject confirmLibrariesPoolsUpload(HttpSession session, JSONObject js
                   ldi.setConcentration(Double.valueOf(jsonArrayElement.getString(11)));
                   ldi.setCreationDate(new Date());
                   ldi.setLastModified(ldi.getCreationDate());
-                  ldi.setDilutionCreator(user.getLoginName());
+                  ldi.setCreator(user);
                   if (!library.getLibraryDilutions().contains(ldi)) {
                     library.addDilution(ldi);
                     log.info("Added library dilution: " + ldi.toString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)